roamance/src/components/Icons.tsx

```
@@ -0,0 +1,126 @@
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width={props.width || "16"}
+      height={props.height || "16"}
+      viewBox="0 0 24 24"
+      fill="none"
+      stroke="currentColor"
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+      {...props}
+    >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)